src/modules/GameConstants.ts

```diff
@@ -346,6 +349,8 @@ export enum Region {
     kalos = 5,
     alola = 6,
     galar = 7,
+    armor = 8,
+    crown = 9,
```

Comment:
Not sure if we should have these as a separate region or not yet, feel like they should just fall under Galar tbh 🤔

And then be stored somewhere else as an extra area along with whatever other stuff isn't in the base games such as `Sevii Islands`